src/modules/AnnotationModule.ts

```diff
@@ -492,7 +492,9 @@ export class AnnotationModule implements ReaderModule {
   public showHighlights() {
```

Comment:
@nikolai-ri i missed this when we spoke, we have to revert the change on the highlights list in this method, as this method is solely responsible for showing all highlights in the toc area , see dita reader example.

Reply:
Thanks & good catch!
I reverted the change in showHighlights to get the annotations of all chapters and can confirm its working in the reader example project.